test/multiverse/suites/rails/Envfile

```diff
@@ -40,7 +40,8 @@ end
 
 if RUBY_VERSION < '3.0.0'
   gemfile <<-RB
-    gem 'rails', '~> 5.2.0'
+    # Rails 5.2.7.1 uses a '~>>' style heredoc that breaks Ruby 2.2 compatibility
+    gem 'rails', '~> 5.2.0'#{", '<= 5.2.7'" if RUBY_VERSION < '2.3.0'}
```

Comment:
Is this still true? If your PR has been merged in, could you remove this?

Reply:
Same goes for: test/multiverse/suites/rails_prepend/Envfile

Reply:
Ah! I've been scanning for a 5.2.7.2 release that never came, but there is a 5.2.8. I'll give it a try!